marimo/_cli/utils.py

```diff
@@ -28,3 +32,171 @@ def prompt_to_overwrite(path: Path) -> bool:
         )
 
     return True
+
+
+def resolve_token(
+    token: bool,
+    token_password: Optional[str],
```

Comment:
nit: you could do `resolve_token(*, ..)` so you have to pass kwargs and don't mix up `token_password` and `token_password_file`

Reply:
Good comment